Makefile

```diff
@@ -35,7 +35,9 @@ build:
 		-o bin/$(BINARY) \
 		github.com/metal-stack/metalctl
 
-	md5sum bin/$(BINARY) > bin/$(BINARY).md5
+	cd bin/; \
```

Comment:
Why ?

Reply:
Because the MD5 hash file contains the wrong path:
```
f423e891ba1034242913b030cc0692c1  bin/metalctl-linux-amd64
```